ncl/receipts/climatology_functions.ipynb

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
<p>Looks like this should be <code>.../ncl_raw/climatology.ncl</code>,not currently populating on the site  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good catch